src/libs/Launcher.js

```diff
@@ -271,9 +270,10 @@ export default class Launcher {
     launcherClient.setAppMetadata({
       sourceAccount: account._id
     })
-    const folder = await ensureKonnectorFolder(client, {
-      konnector,
-      account
+    const folder = await models.konnectorFolder.ensureKonnectorFolder(client, {
+      konnector: { ...konnector, _id: konnector._id },
```

Comment:
Why do you spread?

Reply:
_id attribute is missing in konnector object, which causes the reference to the konnector in the destination folder to be null

I think there is something to fix in harvest when the launcher context is sent

Reply:
I don't understand: 

You do: 
```
      konnector: { ...konnector, _id: konnector._id }
```

and you say : "_id attribute is missing in konnector object,". But you do: `_id: konnector._id` so _id is present in the konnector object. If this is not the case, you'll have `_id: undefined` is it really wanted?

Reply:
Sorry, I miss typed. The right code is `konnector: { ...konnector, _id: konnector.id }` and I fixed it now.